bCNC/CNC.py

```diff
@@ -3362,6 +3362,17 @@ def drill(self, items, depth=None, peck=None, dwell=None, distance=None, number=
 			opname = "drill"
 		else:
 			opname = "drill:%g"%(depth)
+		#Incorrect peck values can block drilling cycle calculation
```

Comment:
Can you please in short explain this to me?

Reply:
Hi Harvie,
   If peck depth is negative input from the user, the pecking is working upwards from the material and not in the correct direction. The peck drilling while loop will never exit. (ln.3442)

Reply:
So why not just take abs() of the value, that would be much shorter code (= easier to maintain). Do you thing that would work?
Currently we have some issues with bCNC maintainability, so i would love to simplify things as much as possible.

Reply:
I was using the more complex style for those who have the stock material settings "upside down".
I can modify to abs() if you prefer that version.

Reply:
We don't really encourage people to use "Z down possitive" coordinate system. In my opinion this is wrong hardware setup and there are lots of things that will be broken in bCNC if you use this. It's like chinese machines comming with Y and Z swapped... Just fix the wiring :-)